maint/local_python/binding_c.py

```diff
@@ -587,6 +694,72 @@ def dump_copy_right():
     G.out.append(" */")
     G.out.append("")
 
+def dump_profiling_qmpi(func, mapping):
+    parameters = ""
+    for p in func['c_parameters']:
+        parameters = parameters + ", " + p['name']
+
+    func_name = get_function_name(func, mapping)
+    func_decl = get_declare_function(func, mapping)
+    qmpi = re.sub(r' MPI', ' QMPI',
+            re.sub(r', \)', ')',
+                re.sub(r'\(', '(QMPI_Context context, int tool_id, ',
+                    re.sub(r'\(void\)', '()',
+                        func_decl, 1),
+                    1),
+                1),
+            1)
```

Comment:
Wrap above to an internal function with a good semantic name. `qmpi` is a bad name, make it more specific.